source/server/main.js

```diff
@@ -453,7 +461,13 @@ function handleInitialization(webSocketConnection, playerUUID) {
   }
 
   const attemptRejoinStatus = attemptRejoin(webSocketConnection, playerUUID);
-  if (!attemptRejoinStatus) createInstance(webSocketConnection);
+  if (!attemptRejoinStatus) {
```

Comment:
an if/else statement should probably have the negation as the else clause instead of the if clause, for clarity's sake